2주차/[Lv.2]_[1차]_뉴스_클러스터링/조은서.js

```diff
@@ -0,0 +1,31 @@
+const makeStr = (str) => {
+  const arr = [];
+  const regExp = /[^a-z]/;
+
+  for (let i = 0; i < str.length - 1; i++) {
+    const tmp1 = str[i].toLowerCase();
+    const tmp2 = str[i + 1].toLowerCase();
+    if (regExp.test(tmp1) || regExp.test(tmp2)) continue;
+    else {
+      arr.push(tmp1 + tmp2);
+    }
+  }
+  return arr;
+};
+
+function solution(str1, str2) {
+  let a1 = makeStr(str1);
+  let a2 = makeStr(str2);
+  let common = 0;
+
+  a1.map((item) => {
```

Comment:
```suggestion
  a1.forEach((item) => {
```
[`map()`](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/Array/map)함수는 새로운 배열은 반환하는 것이고, [`forEach()`](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach)함수는 각 요소를 순환하는 것(?)이라고 할 수 있기 때문에 이 상황에서는  `forEach()`를 쓰시는 것이 맥락에 맞는 것 같습니다!! 😆